src/tools/flash_opts.c

```diff
@@ -11,6 +11,18 @@ static bool starts_with(const char * str, const char * prefix) {
     return (0 == strncmp(str, prefix, n));
 }
 
+static int invalid_args(const char *expected)
+{
```

Comment:
May you adjust the code style here and the one of `bad_arg`? It looks a little inconsistent considering the context.

Reply:
Do you mean the { at the end of the line ?

Reply:
Exactly

Reply:
Hopefully fixed

Reply:
Thx :-)